.env.example

```
@@ -16,6 +16,13 @@ JWT_ADMIN_ACCESS_SECRET=your-admin-access-secret-key
 JWT_ADMIN_REFRESH_EXPIRATION=7d
 JWT_ADMIN_REFRESH_SECRET=your-admin-refresh-secret-key
 
+# Google OAuth (Google Cloud Console에서 발급)
+GOOGLE_CLIENT_ID=your-google-client-id
+GOOGLE_CLIENT_SECRET=your-google-client-secret
+GOOGLE_CALLBACK_URL=http://localhost:3000/v1/auth/google/callback
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GOOGLE_CALLBACK_URL key should go before the GOOGLE_CLIENT_ID key

(UnorderedKey)